lib/src/pages/splash.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:help_a_paw/src/pages/home.dart';
+import 'package:help_a_paw/src/pages/login.dart';
+import '../controllers/user_controller.dart';
+import '../services/constants.dart' as global;
+
+class SplashScreen extends StatelessWidget {
+  const SplashScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    SystemChrome.setPreferredOrientations([
+      DeviceOrientation.portraitUp,
+    ]);
+
+    Future.delayed(const Duration(seconds: 2), () async {
+      bool isLoggedIn = await UserController().isUserLoggedIn;
+      Navigator.of(context).push(
+        MaterialPageRoute(
+            builder: (context) => isLoggedIn ? const HomePage(title: 'Help a Paw') : const LoginPage(),
```

Comment:
Actually we don't want to prevent users from viewing signals when not logged in. It's required only for active operations (e.g. submit a signal, comment, etc)

Reply:
ok. so before submitting the signal we have to show the alert please log in and then proceed right?

Reply:
Yes, exactly.

Reply:
Ok i will change